docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+Camus
+================
+
+Camus is a pipeline from Kafka to HDFS.
+It is a MapReduce job that does distributed data loading from Kafka to HDFS.
+At LinkedIn, Camus is used to process tens of billions of messages per day.
+The confluent Camus distribution supports the latest CDH distribution.
+
+Camus includes the following features:
+    #. Automatic topic discovery
+    #. Avro schema management
+    #. Date partitioning
+
+All work is done within a single MapReduce job divided into three stages:
+    #. Setup stage:
+        Fetch available topics and partitions from Zookeeper and latest offsets from Kafka.
+    #. Hadoop stage:
+        Allocates topic pulls among a set number of MapReduce tasks. Each task does the following:
+            #. Fetch events from Kafka server and collect count statistics.
+            #. Write data to corresponding directories based on timestamps of events.
+            #. Produce count events and write to HDFS.
+            #. Store updated offsets in HDFS.
+    #. Cleanup stage:
+        Read counts from all tasks, aggregates the values,
+        and submits the results to Kafka for consumption by other systems.
+
+Quickstart 
+----------
+
+Assuming you have confluent platform installed.
+
+.. sourcecode:: bash
+
+   # Assuming that a Hadoop cluster is already deployed and you have access to it
+   # Start a testing local Kafka cluster (1 Zookeeper node, 1 Kafka node)
+   $ cd kafka
+   $ bin/zookeeper-server-start.sh config/zookeeper.properties
+   $ bin/kafka-server-start.sh config/server.properties
+   # Create testAvro topic in Kafka
+   $ bin/kafka-topics.sh --create --zookeeper localhost:2181 \
+         --topic testAvro --partitions 1 --replication-factor 1
+   $ cd ..
+
+   # Start Schema Registry. The default schema registry settings
+   # automatically work with the default settings of ZooKeeper and Kafka.
+   $ cd schema-registry
+   $ bin/schema-registry-start config/schema-registry.properties
+   $ cd ..
+
+   # Start Kafka rest proxy.
+   $ cd kafka-rest
+   $ bin/kafka-rest-start kafka-rest.properties
+   $ cd ..
+
+   # Assuming that you have hadoop on your PATH, and make sure that
+   # schema.registry.url points to the correct address
+   $ cd camus
+   $ hadoop jar confluent-camsu-0.1.0-SNAPSHOT-shaded.jar com.linkedin.camus.etl.kafka.CamusJob \
+     -P etc/camus.properties
```

Comment:
Typo: confluent-camsu-0.1.0-SNAPSHOT-shaded.jar -> confluent-camus-0.1.0-SNAPSHOT-shaded.jar. Also, assume we're going to tag this as 1.0 and update the version numbers.

Finally, I took a look at the avro-decoder branch and it looks like camus.properties isn't using our decoder class, which means as it stands this quickstart won't work. I mentioned it in the PR for the new decoder since fixing it there will resolve the problem here.